tests/parse-link-header.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+const parseLinkHeader = require('../lib/parse-link-header');
+const assert = require('assert');
+
+describe('Parse Link Header', () => {
+    it('returns uri and rel of the passed header', () => {
+        const linkHeader =
+            '<http://a.com/app.js>; rel="script",<http://a.com/app.css>; rel="stylesheet"';
+
+        assert.deepStrictEqual(parseLinkHeader(linkHeader), [
+            { uri: 'http://a.com/app.js', rel: 'script' },
+            { uri: 'http://a.com/app.css', rel: 'stylesheet' }
+        ]);
+    });
+
+    it('ignore attributes other than rel and uri', () => {
+        const linkHeader =
+            '<http://a.com/app.js>; rel="script"; crossorigin="anonymous"';
+
+        assert.deepStrictEqual(parseLinkHeader(linkHeader), [
+            { uri: 'http://a.com/app.js', rel: 'script' }
+        ]);
+    });
+
+    it('filters invalid header links', () => {
+        const linkHeader = 'http://a.com/app.js; rel="script"';
+
+        assert.deepStrictEqual(parseLinkHeader(linkHeader), []);
+    });
+
+    it('filters invalid rel attributes', () => {
```

Comment:
What is an invalid rel attr?
Empty? Can I use in rel="styleshit"?

Reply:
Ofcourse you can lol. its valid. But wont be loaded because of the check in getFragmentAssetUris.

Reply:
But the test if for rel="" and no rel attribute at all. do you have any better naming suggestion if its confusing?

Reply:
maybe it could be rephrased... but it is fine.